python_modules/dagster/dagster/_cli/definitions.py

```diff
@@ -43,11 +43,11 @@ def definitions_cli():
     help="Load the code locations using a gRPC server, instead of in-process.",
 )
 @click.option(
-    "--verbose",
+    "--verbose-stack-traces",
     "-v",
     flag_value=True,
     default=False,
-    help="Show verbose error messages, including system frames in stack traces.",
+    help="Show verbose stack traces, including system frames in stack traces.",
```

Comment:
feel like most cli tools just have a single verbose flag that turns up everything, might be simpler than having fine grain verbosity 

did we conflict with some other notion of verbose?

Reply:
same feeling. i liked `--verbose` better

Reply:
changed back to `--verbose` 🫡